SpeechResponder/eddi.json

```diff
@@ -461,6 +461,14 @@
       "name": "Docking timed out",
       "description": "Triggered when your docking request times out"
     },
+    "Empire promotion": {
+      "enabled": true,
+      "priority": 3,
+      "responder": true,
+      "script": "\"{if event.rank = \\\"Knight\\\":\\r\\n    sir\\r\\n|elif event.rank = \\\"Lord\\\":\\r\\n    my lord\\r\\n|elif event.rank = \\\"Baron\\\":\\r\\n    my lord\\r\\n|elif event.rank = \\\"Viscount\\\":\\r\\n    my lord\\r\\n|elif event.rank = \\\"Count\\\":\\r\\n    my lord\\r\\n|elif event.rank = \\\"Earl\\\":\\r\\n    my lord\\r\\n|elif event.rank = \\\"Marquis\\\":\\r\\n    my lord\\r\\n|elif event.rank = \\\"Duke\\\":\\r\\n    your grace\\r\\n|elif event.rank = \\\"Prince\\\":\\r\\n    your royal highness\\r\\n|elif event.rank = \\\"King\\\":\\r\\n    your majesty\\r\\n|else:\\r\\n    commander\\r\\n},\\r\\nthe Empirial Navy has \\r\\n{OneOf(\\\"granted you a promotion\\\", \\\"promoted you\\\", \\\"advanced you\\\", \\\"named you\\\")}\\r\\nto the rank of\\r\\n{event.rank}\\r\\n{Occasionally(2, \\\"for your service to the Empire\\\")}.      \"default\": true,",
```

Comment:
I build this branch and found that the Empir promotion script has gotten garbled. Looks like it has been double-escaped for JSON. Look at it in the app and you'll see.

Reply:
Done. Not sure how that happened?